DDG4/src/Geant4PhysicsList.cpp

```diff
@@ -362,6 +365,14 @@ G4VUserPhysicsList* Geant4PhysicsListActionSequence::extensionList()    {
   // Ownership is transferred to the physics list.
   // Do not delete this pointer afterwards....
   physics->RegisterPhysics(new ParticlePhysics(this,physics));
+
+  //Setting verbosity for pieces of the physics
+  physics->SetVerboseLevel(m_verbosity);
+  if(G4EmParameters::Instance())
```

Comment:
These checks make no sense, because they will just initialise if not initialised?